src/com/company/Board.java

```diff
@@ -0,0 +1,100 @@
+package com.company;
+
+import java.util.Arrays;
+
+public class Board implements IBoard {
+
+  private final int Row;
+  private final int Column;
+  String[][] board;
+
+  public Board(int row, int column) {
+    Row = row;
+    Column = column;
+    board = new String[Row][Column];
+    SetBoard(board);
```

Comment:
function name should be lowercase. You already know that so why starting with capital letter?

Reply:
ok, will fix that